cmake/BLTSetupTargets.cmake

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+# This file is intended to be included in the *-config.cmake files of
+# any project using a third-party library.  The macro 
+# `blt_install_tpl_setups(DESTINATION <dir>)`  installs this file
+# into the destination specified by the argument <dir>.
+
+# BLTInstallableMacros provides helper macros for setting up and creating
+# third-party library targets.  The below guard prevents the file from 
+# included twice when a project builds using BLT.
+if (NOT BLT_LOADED)
+  include("${CMAKE_CURRENT_LIST_DIR}/BLTInstallableMacros.cmake")
+endif()
+# If the generated TPL config file exists, include it here.
+if (EXISTS "${CMAKE_CURRENT_LIST_DIR}/BLTThirdPartyConfigFlags.cmake")
+  include("${CMAKE_CURRENT_LIST_DIR}/BLTThirdPartyConfigFlags.cmake")
+# Otherwise, configure the TPL flags.  We have to prefix these variable with
+# BLT so that they never conflict with user-level CMake variables in downstream
+# projects.
+else()
```

Comment:
```suggestion
else()
  # Otherwise, configure the TPL flags.  We have to prefix these variable with
  # BLT so that they never conflict with user-level CMake variables in downstream
  # projects.
```